README.md

```diff
@@ -18,11 +18,11 @@ A CLI for [Groonga](http://groonga.org/) using Readline. Supports Windows.
 
 For example:
 
-    $ grnlip groonga ~/.milkode/db/milkode.db
+    $ grnlip groonga %HOME%\.milkode\db\milkode.db
```

Comment:
I think this change is needless because the line is for Linux. See also the prompt.